name-segments/nouns/aquatic.txt

```diff
@@ -0,0 +1,42 @@
+lake
+pond
+ocean
+basin
+flood
+hail
+cove
+beach
+ice
+snow
+sleet
```

Comment:
could you move these lines into "tundra.txt", as they're probably more appropriate there.